cookbook/integrations/flytekit_plugins/dolt/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM python:3.8-buster
+
+WORKDIR /root
+ENV VENV /opt/venv
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV PYTHONPATH /root
+
+# Install dolt
+RUN sudo bash -c 'curl -L https://github.com/dolthub/dolt/releases/latest/download/install.sh | sudo bash' &&\
+    dolt config --global --add user.email bojack@horseman.com &&\
+    dolt config --global --add user.name "Bojack Horseman" &&\
+    dolt config --global --add metrics.host eventsapi.awsdev.ld-corp.com &&\
```

Comment:
can we not add this? sorry but otherwise we will have to ask for permissions

Reply:
Is it OK if this is all hand-tested only? You need dolt to run any of these examples, and auto-tests will also need dolt

Reply:
what i meant was the just eventsapi